# Stefan Teleman <stefan.teleman@oracle.com>
# Port of CilkPlus (libcilkrts) to SPARC andd Solaris.
# How did this ever compile. There is no variable named 't'.
# There is, however, a variable named 'sf', and that's the one
# we want.
# Internal patch. SPARC specific:
# https://gcc.gnu.org/bugzilla/show_bug.cgi?id=68945
--- libcilkrts/runtime/config/generic/cilk-abi-vla.c	2013-10-29 11:37:47.000000000 -0700
+++ libcilkrts/runtime/config/generic/cilk-abi-vla.c	2015-08-05 18:16:04.959499239 -0700
@@ -103,5 +103,5 @@
 
     // Just free the allocated memory to the heap since we don't know
     // how to expand/contract the calling frame
-    vla_internal_heap_free(t, full_size);
+    vla_internal_heap_free(sf, full_size);
 }
